docs/admin.rst

```diff
@@ -183,18 +180,23 @@ SecureDrop. Make sure the journalist remembers their username and
 passphrase and always has their 2 factor authentication device in their
 possession when they attempt to log in to SecureDrop.
 
-.. |SecureDrop main page|
-  image:: images/manual/screenshots/journalist-admin_index_no_documents.png
-.. |SecureDrop admin home|
-  image:: images/manual/screenshots/journalist-admin_interface_index.png
-.. |Add a new user|
-  image:: images/manual/screenshots/journalist-admin_add_user_totp.png
-.. |Enable FreeOTP|
-  image:: images/manual/screenshots/journalist-admin_new_user_two_factor_totp.png
-.. |Enable YubiKey|
-  image:: images/manual/screenshots/journalist-admin_add_user_hotp.png
-.. |Verify YubiKey|
-  image:: images/manual/screenshots/journalist-admin_new_user_two_factor_hotp.png
+
+Passphrases and Two-Factor Resets
+---------------------------------
+
+While we publish some :ref:`passphrase best practices <passphrase_best_practices>`,
+your journalists may occasionally lock themselves out of their accounts. This
+can happen if, for example, they lose their two-factor device or if they
+forget the passphrase  to their password manager. When this happens, you
+can reset  their account by logging in as an administrator to the
+*Journalist Interface* and selecting *Admin* at the top right. Find their
+username and select **Edit**. Next, you can either rotate their passphrase or
+reset their two-factor authentication. You should make sure the user's
+passphrase is saved in a password manager *before* selecting "Reset Password".
+There are two options for resetting two-factor authentication depending on it
```

Comment:
"There are two options for resetting two-factor authentication depending on it was configured during on-boarding."

I'm not sure what you mean here, I must admit.